src/plugins/intel_gpu/src/plugin/ops/loop.cpp

```diff
@@ -30,8 +30,14 @@ namespace intel_gpu {
 template<class DATA_TYPE>
 static DATA_TYPE CreateScalarData(ProgramBuilder &p, const cldnn::primitive_id& id, ov::Shape& shape, cldnn::data_types dtype, int64_t num, int64_t rank) {
     auto mem = p.get_engine().allocate_memory({ shape, dtype, cldnn::format::get_default_format(rank) });
-    cldnn::mem_lock<int64_t> ptr{mem, p.get_engine().get_service_stream()};
-    *ptr.begin() = num;
+    if ( dtype == cldnn::data_types::i32 ) {
```

Comment:
Please stick to an existing code style
```suggestion
    if (dtype == cldnn::data_types::i32) {
```

Reply:
done